### DISCUSSION TITLE
PCL II应该增加我的世界服务器版块

### DISCUSSION THREAD

**Discussion by HexianTao9998** [Meloong-Git/PCL#1535] [讨论]
PCL II是一个非常受欢迎的游戏社区，拥有众多玩家。然而，该社区并没有专门的我的世界服务器版块，这让很多玩家感到失望。

我的世界是一款全球知名的游戏，具有极高的自由度和互动性。在我的世界中，玩家可以建造自己的世界，与其他玩家交流和合作。它已经成为了一个庞大而精彩的游戏社区，并吸引了无数的玩家。

因此，PCL II应该增加我的世界服务器版块。这将为玩家提供一个更加全面和多元化的游戏体验，并吸引更多的玩家来加入这个社区。

首先，增加我的世界服务器版块将扩大PCL II的受众群体。许多喜欢我的世界的玩家会更倾向于加入有关该游戏的社区。如果PCL II能够提供高质量的我的世界服务器，这将肯定吸引更多的玩家加入该社区。这将进一步提高该社区的活跃程度和影响力。

其次，增加我的世界服务器版块也将为现有的PCL II玩家提供更丰富的游戏体验。许多PCL II的玩家也是我的世界的忠实拥趸，因此他们会渴望在该社区中找到关于这个游戏的资源和交流。通过增加我的世界服务器版块，这些玩家将能够更好地进行互动和合作。

最后，增加我的世界服务器版块也可以为PCL II带来商业机会。我的世界是一个高度流行的游戏，有很多玩家会为了获得更好的体验而愿意花费钱购买虚拟物品或服务。如果PCL II能够提供高品质的我的世界服务器，这将为该社区赚取一定的收入，并帮助其发展壮大。

综上所述，PCL II应该增加我的世界服务器版块。这将为其吸引更多的玩家、提高现有玩家的游戏体验，以及为其带来商业机会。如果PCL II能够照顾好这个新建的版块，那么这将是对该社区的巨大发展推动。

**Comment by WForst-Breeze** [comment]
#1047 #706 #340 #321 类似反馈，已被拒绝
- 联机模块现在仍在制作中，但社区服务器和社区云端服务不会制作，因为这会大大增加 PCL2 的维护成本，并需要抵御网络攻击。
- 另一方面，也是重要的一点，内置服务器会进一步触及一部分组织的利益，并引起更多不必要的麻烦。
- PCL2 本质来讲是启动器，而非开服工具，这已经大大超过了 PCL2 的范畴
- 如果制作该功能，会大大拖慢其余功能的制作进度，以进一步拖慢 PCL2 更新速度
- PCL2 本质是**免费的** Minecraft 启动器，一旦进行商业化，同样会引来不必要的麻烦或是法律问题

感谢您对 PCL2 发展的支持和理解

**Reply by iPodtouch6557** [reply]
bakaxl笨蛋广场只是下载的，别管它

**Reply by SALTWOOD** [reply]
我觉得可以把服务交给社区的力量（？
就是PCL2启动器允许添加第三方的服务器列表服务

**Reply by WForst-Breeze** [reply]
> 我觉得可以把服务交给社区的力量（？
> 就是PCL2启动器允许添加第三方的服务器列表服务

#1047

**Comment by iPodtouch6557** [comment]
还有，以后说话言简意赅一点，我们可以理解你想添加该功能（事实上我也认为这个功能是很好的），但是我认为你说话有点不必要的话语……这么长的玩意我要是心血来潮能看完的概率大约相当于中彩票（）
~（有没有可能是我说话习惯是极简风格所以我不想看很长的文字~
edit：如果你是用ChatGPT生成的，当我没说
